pkg/events/derive/derive.go

```diff
@@ -132,6 +132,8 @@ func buildDerivedEvent(baseEvent *trace.Event, skeleton deriveBase, argsValues [
 		return trace.Event{}, unexpectedArgCountError(skeleton.Name, len(skeleton.Params), len(argsValues))
 	}
 	de := *baseEvent
+	// TODO: this logic must be accessible from the policy manager
```

Comment:
Which logic? The deriving from a base? Asking so that the comment's instructions can be more clear.

Reply:
Gonna improve this.